site/src/components/pages/user/list-view-card.tsx

```diff
@@ -1,107 +1,103 @@
-import { Box, Typography } from "@mui/material";
```

Comment:
On this file, basically added `extraContent` instead of `lastUpdated` for customizability. And removed the extra `Box` wrapping everything.